frontend/src/views/Login.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import './NewStore.scss';
+import Button from "components/Button/Button";
+import Form from "components/Form/Form";
+import ViewWrappers from "components/ViewWrappers/ViewWrappers";
+import { useUserContext } from '../context/UserContext';
+import API from '../shared/api';
+
+const Login = ({history}) => {
+
+  const { setUser, setVerified } = useUserContext();
+
+  const [user, updateUser] = useState({
+    username: "",
+    password: ""
+  })
+
+  const changeHandler = e => {
+    let _user = {...user};
+    _user = { [e.target.name]: e.target.value}
+    updateUser({...user, ..._user})
+  }
+
+  const submitHandler = async (e) => {
+    e.preventDefault();
+    const getToken = await API.authToken.post({...user})
+    let token = getToken.data.token
+
+    const getUser = await API.users.get()
+    let currentUser = getUser.data.find(x => x.username === user.username )
+
+    if(token){
+      setUser(currentUser, token)
+      setVerified(true)
+      // for now the user gets redirected to /stores, should be redirected to
+      // order views
+      history.push('/stores/');
+    }
+    else {
+      console.error(getToken)
+    }
+  }
+  return (
+    <ViewWrappers.View withPadding>
+          {/* initial view */}
+            <Form
+              body= { <>
+                  <label> E-mail Addresse:
+                    <input onChange={changeHandler} type="email" id="email" name="username"/>
```

Comment:
Please map the email to the username when you post to the API and not in the frontend, it might mess with autocompletion etc.